oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/Event.java

```diff
@@ -99,7 +99,7 @@ protected String id0() {
     @Column(columnName = MESSAGE)
     private String message;
 
-    @Column(columnName = PARAMETERS, storageOnly = true, length = 1024)
+    @Column(columnName = PARAMETERS, storageOnly = true, length = 20000)
```

Comment:
I think we need to control length in serialization process too? Otherwise still could have over length.